projects/go-lib/src/lib/components/go-select/go-select.component.spec.ts

```diff
@@ -102,4 +102,19 @@ describe('GoSelectComponent', () => {
       expect(component.control.value).toEqual([1, 2, 3]);
     });
   });
+
+  describe('onRemoveAll()', () => {
+    it('uses bindValue to get value if bindValue exists', () => {
```

Comment:
We should change the description here to be something related to what we're testing.

Reply:
Done the changes